templates/page.html.twig

```diff
@@ -71,7 +71,7 @@
     <main role="main" {{ attributes.addClass(layout_classes) }}>
       <a id="main-content" tabindex="-1"></a>{# link is in html.html.twig #}
 
-      {% if page.sidebar_first|render %}
+      {% if page.sidebar_first|render|striptags|trim is not empty %}
         <aside class="cd-layout-sidebar-first" role="complementary">
```

Comment:
Ensuring the sidebars are rendering when desired is proving difficult.
See https://www.drupal.org/node/953034 for history. This is an ongoing Drupal issue.
When there are blocks in regions that are access controlled (eg. Tools block, displays for admins when logged in), the empty region div with print for anon users.
When I add filters `|striptags|trim is not empty ` to `page.sidebar_second|render` the Facets in the sidebars don't render (i.e. no sidebar prints). The only results where I currently get Facets printing in sidebar (for either anon and auth) is to use `page.sidebar_second|render` only